operator/v1/types_ingress.go

```diff
@@ -125,6 +127,12 @@ type IngressControllerSpec struct {
 	//
 	// +optional
 	NodePlacement *NodePlacement `json:"nodePlacement,omitempty"`
+
+	// tlsSecurityProfile specifies settings for TLS connections for externally exposed servers.
```

Comment:
"servers"→"IngressControllers", or for better consistency in terminology, how about "externally published IngressControllers"?

Reply:
Fixed